### ISSUE TITLE
Partitions not mounted after installation on Windows 10 21H1

### ISSUE THREAD

**Issue by daviessm** [maharmstone/btrfs#388]
After installing the driver using right click on btrfs.inf and "install" then rebooting, I don't see my drive mounted in This PC in Windows 10.

    C:\Users\steved>sc query btrfs

    SERVICE_NAME: btrfs
        TYPE               : 1  KERNEL_DRIVER
        STATE              : 4  RUNNING
                                (STOPPABLE, NOT_PAUSABLE, IGNORES_SHUTDOWN)
        WIN32_EXIT_CODE    : 0  (0x0)
        SERVICE_EXIT_CODE  : 0  (0x0)
        CHECKPOINT         : 0x0
        WAIT_HINT          : 0x0

Device Manager shows this:
![Capture](https://user-images.githubusercontent.com/3835494/121507917-fbd59a00-c9dc-11eb-8d69-cd6f47ee7d03.PNG)

What have I missed?

**Comment by blackcat-917** [comment]
Try with the [driver provided in the releases page](https://github.com/maharmstone/btrfs/releases/tag/v1.7.8#:~:text=btrfs-1.7.8.zip), the chocolatey package didn't work for me

**Comment by Obi001** [comment]
Maybe this will fix the issue being discussed:

disconnect the device
    Open a Command Prompt ran as Administrator.
    Run diskpart.
    Type automount enable and press Enter.
    Type exit and press Enter.
    Close the Command Prompt window.
connect the device again

**Comment by daviessm** [comment]
I played around with `diskpart` and `automount` was already enabled. What I had to do in the end was use `mountvol` to mount the drive then it stays mounted after a reboot also: `mountvol /?` to show available volumes then `mountvol B: \\?\<guid>\` to mount the partition.

Unfortunately I can't find a way of mapping GUIDs to partitions in Windows so it was a case of trial and error to get to that point - of course it was the last volume in the list.
